Source/WebKit/Resources/SandboxProfiles/ios/com.apple.WebKit.WebContent.sb.in

```diff
@@ -212,6 +212,7 @@
                 "IOAccelDevice2"
                 "IOAccelSharedUserClient2"))
 
+#if !PLATFORM(IOS)
```

Comment:
IOS, and not IOS_FAMILY?

Reply:
> IOS, and not IOS_FAMILY?

Yes, I think we still would like to be able to send these extensions for other platforms in the iOS family, IIRC.

Thanks for reviewing!